src/MainWindow.cpp

```diff
@@ -721,6 +721,20 @@ void MainWindow::populateTable()
         connect(ui->dataTable->selectionModel(), SIGNAL(currentChanged(QModelIndex,QModelIndex)), this, SLOT(dataTableSelectionChanged(QModelIndex)));
         connect(ui->dataTable->selectionModel(), &QItemSelectionModel::selectionChanged, [this](const QItemSelection&, const QItemSelection&) {
             updateInsertDeleteRecordButton();
+
+            const QModelIndexList& sel = ui->dataTable->selectionModel()->selectedIndexes();
+            if (sel.count() > 1 && sel.count() < Settings::getValue("databrowser", "complete_threshold").toInt()) {
```

Comment:
Could we move the part about the numbers of rows and columns out of this if statement? Unlike sum and average, they don't seem to cause any performance problems even for large selections. And they are definitely nice to have :smile:

Reply:
Yes, that makes sense. I'll do that.